docs/code-pr-advice.md

```diff
@@ -79,7 +79,7 @@ a "`BUG: feature X not implemented see {bug-url}`" type error.
 - Don't use multiple log calls when a single log call could be used.
 
 - Use structured logging where possible to allow
-  [standard tooling](https://github.com/kata-containers/tests/tree/main/cmd/log-parser)
+  [standard tooling](https://github.com/kata-containers/kata-containers/tree/main/src/tools/log-parser)
```

Comment:
I think it would be nicer to use relative links whenever possible

Reply:
I agree with you that the priority is to use relative paths